### PR TITLE
Fix exporting conversations

### PR DESCRIPTION
**Pull request by lferran**
### Description
Since conversations are stored across several pages in different maindb keys, the `get_value` field does not really work when generating the broker message to export a resource.

### How was this PR tested?
Integration tests

### CODE REVIEW

File: nucliadb/nucliadb/ingest/tests/integration/orm/test_orm_resource.py [hunk 1]
```diff
@@ -215,3 +218,140 @@ async def test_vector_duplicate_fields(
...
+    async with maindb_driver.transaction() as txn:
+        r.txn = txn
+        bm = await r.generate_broker_message()
+
+    # Check generated broker message has the same metadata as the created resource
+
+    # 1. ROOT ELEMENTS
+    # 1.1 BASIC
+    basic = bm.basic
+    assert basic.HasField("created")
```
Comment (tarekziade) [inline nucliadb/nucliadb/ingest/tests/integration/orm/test_orm_resource.py:244]: that's a huge list of asserts. I wonder if you could create a `basic` object and assert it just once?

**Review by tarekziade — APPROVED** [review]
just a question

File: nucliadb/nucliadb/ingest/tests/integration/orm/test_orm_resource.py [hunk 2]
```diff
@@ -94,13 +97,13 @@ async def test_create_resource_orm_with_basic(
     o2 = PBOrigin()
     assert o2 is not None
     o2.source = PBOrigin.Source.API
-    o2.source_id = "My Surce"
+    o2.source_id = "My Suorce"
```
Comment (jotare) [inline nucliadb/nucliadb/ingest/tests/integration/orm/test_orm_resource.py:100]: You meant, `Source`?

File: nucliadb/nucliadb/ingest/tests/integration/orm/test_orm_resource.py [hunk 3]
```diff
@@ -94,13 +97,13 @@ async def test_create_resource_orm_with_basic(
...
     o2.created.FromDatetime(datetime.now())
 
     await r.set_origin(o2)
     o2 = await r.get_origin()
     assert o2 is not None
-    assert o2.source_id == "My Surce"
+    assert o2.source_id == "My Suorce"
```
Comment (jotare) [inline nucliadb/nucliadb/ingest/tests/integration/orm/test_orm_resource.py:106]: Same

**Review by jotare — APPROVED** [review]

File: nucliadb/nucliadb/ingest/tests/integration/orm/test_orm_resource.py (hunk 1 above)
Comment (lferran) [reply nucliadb/nucliadb/ingest/tests/integration/orm/test_orm_resource.py:244]: I know... the problem is that the basic has some datetime fields that will make the assertion false :(

**Pull request merged**